lib/senkyoshi/models/gradebook.rb

```diff
@@ -23,5 +39,29 @@ def get_categories(data)
         categories[id] = title
       end
     end
+
+    def get_outcome_definitions(xml)
+      xml.xpath("//OUTCOMEDEFINITION").map do |outcome_def|
+        category_id = outcome_def.xpath("CATEGORYID/@value").first.value
+        OutcomeDefinition.from(outcome_def, @categories[category_id])
+      end
+    end
+
+    def convert_categories(course)
+      @categories.each do |category|
+        if AssignmentGroup.find_group(course, category.last).nil?
+          course.assignment_groups <<
+            AssignmentGroup.create_assignment_group(category.last)
+        end
+      end
+    end
+
+    def canvas_conversion(course, _ = nil)
```

Comment:
if the param is going to be used, don't use `_` to define it. Call it `resources`